www/src/Home.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { Link, Route, Switch } from 'react-router-dom';
+import Jumbotron from './components/Jumbotron';
+import daniel from './daniel.jpg';
+
+import {
+  Container,
+  Grid,
+  Header,
+  List,
+  Card,
+  Image,
+  Button,
+  Segment,
+  Menu,
+} from 'semantic-ui-react'
+
+
+const themes = [
+  'Massively',
+  'Stellar',
+  'Photon',
+  'Eternal',
+  'Joyful',
+  'Lens',
+]
+
+const Home = () => (
+    <div>
+      <Grid celled='internally' columns='equal' stackable>
+
+        <Grid.Row textAlign='center'>
+          <Grid.Column>
+            <Jumbotron>
+              <Header as='h1'>
+                Themes for Gatsby
+                <Header.Subheader>
+                  Easy to setup. Easy to customize. Easy to deploy.
+                </Header.Subheader>
+              </Header>
+            </Jumbotron>
+          </Grid.Column>
+        </Grid.Row>
+
+        <Grid.Row>
+          <Grid.Column>
+            <Card.Group centered itemsPerRow={3}>
```

Comment:
Can make this short hand to eliminate boilerplate code. Pass and items prop with all the data in objects that contain the props of the individual card component.